kivy/core/image/__init__.py

```diff
@@ -830,10 +833,23 @@ def save(self, filename, flipped=False):
             Parameter `flipped` added to flip the image before saving, default
             to False.
 
+        .. versionchanged:: 1.11.0
+            Parameter `fmt` added to force the output format of the file
+            Filename can now be a BytesIO object.
+
         '''
+        if isinstance(filename, BytesIO):
+            is_bytesio = True
+            if not fmt:
+                raise Exception("You must specify a format to save into a BytesIO object")
```

Comment:
is this covered by the unit tests?

Reply:
Ok thanks! This what I was suspecting. Then this is usually covered with `assertRaises` not try/catch. Because it makes things more obvious/explicit. I will take another look once on laptop